openedx/core/djangoapps/programs/tasks/v1/tasks.py

```diff
@@ -122,6 +123,9 @@ def award_program_certificates(self, username):
         None
 
     """
+    if configuration_helpers.get_value('partner_without_program_certificates', False):
```

Comment:
I think single configuration `programs_without_certificates ` should be used for both cases. For all programs it should be something like this `programs_without_certificates = ["ALL"]` and for selected programs `programs_without_certificates = ["program uuid","program uuid" ...]`